website/app/components/doc/token-preview/index.hbs

```diff
@@ -0,0 +1,22 @@
+{{!
```

Comment:
I've abstracted away the "Token Preview" component under the assumption that I could re-use it in the search results too. In reality is not possible, but since it likely will come handy in the future, I've left it here. Let me know if you prefer me to move it to a separate PR.